CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2024-12-23 -- v1.1.0
+### Added
+- Re-query ShopperTrak API for sites missing from a previous API response. This is distinct from re-querying for sites that appear in the API response but have unhealthy data (which is already done).
```

Comment:
What spurred this change? It makes sense to me why you're doing it, was just curious -- did STRAD point out a discrepancy or was this something you just noticed?

Reply:
Something I noticed -- one day there were way fewer sites than usual in Redshift and we weren't retrying the API because we were only checking for unhealthy data not completely missing data